notebooks/en/chain_of_verification.ipynb

```diff
@@ -0,0 +1,470 @@
+{
```

Comment:
<p>It would be helpful to explain the config a little bit here. At least its essential components. Maybe add some advice on the structure/content of the prompts?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/29/discussion/#comment-1494821409'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Added a description and links